app1.js

```diff
@@ -0,0 +1,35 @@
+// Get the modal
+
+const modal = document.getElementsByClassName('myModal');
+const btn = document.getElementsByClassName('myBtn');
+const span = document.getElementsByClassName('close');
+
+function setDataIndex() {
+  for (let i = 0; i < btn.length; i + 1) {
+    btn[i].setAttribute('data-index', i);
+    modal[i].setAttribute('data-index', i);
+    span[i].setAttribute('data-index', i);
+  }
+}
+
+window.onload = function () {
+  setDataIndex();
+};
+
+for (let i = 0; i < btn.length; i + 1) {
+  btn[i].onclick = function () {
+    const ElementIndex = this.getAttribute('data-index', i);
+    modal[ElementIndex].style.display = 'block';
+  };
+
+  span[i].onclick = function () {
+    const ElementIndex = this.getAttribute('data-index', i);
+    modal[ElementIndex].style.display = 'none';
+  };
+}
```

Comment:
- [ ] According to the instructions, you are supposed to use a javascript object to store information for all projects as seen here. Please fix that 👍  
![image](https://user-images.githubusercontent.com/45460192/175527532-91e198ab-00e9-40aa-bb3c-f6656bfed000.png)